### DISCUSSION TITLE
NutUI React 适配Taro 1.3.4 版本问题汇集

### DISCUSSION THREAD

**Discussion by xiaoyatong** [jdf2e/nutui-react#314] [Q&A]
可以在这里把问题，粘贴在这里。

**Comment by xiaoyatong** [comment]
![image](https://user-images.githubusercontent.com/84436086/191638844-78e2f82a-d171-4a9b-bee7-d5b8b96fb9aa.png)

**Comment by xiaoyatong** [comment]
![image](https://user-images.githubusercontent.com/84436086/191639653-429ff793-0c8c-40e0-83f1-84595bcce0e5.png)

**Comment by xiaoyatong** [comment]
![image](https://user-images.githubusercontent.com/84436086/191639715-b374e044-4d42-4984-b337-b749991c9185.png)

**Comment by xiaoyatong** [comment]
滚动穿透问题
![image](https://user-images.githubusercontent.com/84436086/191639744-24fa281a-17eb-4c5e-b4e6-66e05ebb05bf.png)

**Reply by xiaoyatong** [reply]
iPhone12promax  iOS16.0  微信8.0.88

**Reply by xiaoyatong** [reply]
8.0.28

**Comment by xiaoyatong** [comment]
没有实际效果
![image](https://user-images.githubusercontent.com/84436086/191639781-29060c15-288f-4f8c-9093-4e8afb6d8f2e.png)

**Comment by xiaoyatong** [comment]
这些动画都调整一下，太生硬了:menu、range、collapse

**Comment by xiaoyatong** [comment]
滚动穿透默认Overlay组件锁定背景就好了，可是我们的Overlay组件却要手动设置是否锁定背景
但是弹框的组件内调用了Overlay组件，就没办法手动控制了
默认true多好

**Comment by xiaoyatong** [comment]
有人弄过表情包组件吗？类似掘金的评论那个。

**Comment by crazyshakalaka** [comment]
Menu 展示card支出相应宽度
![c82e4243fe39695386d3fb38fe96ac6](https://user-images.githubusercontent.com/46771139/191640033-ffe7493c-ca1f-4512-be85-574461d6c8ea.jpg)

**Comment by xiaoyatong** [comment]
![image](https://user-images.githubusercontent.com/84436086/191640025-3e9d2a47-b351-40be-ba2c-986c9038af7f.png)
未完成适配的demo例子，直接隐藏掉，会不会更好一些。
推荐拿掉，加到未来计划里[旺柴]

**Comment by xiaoyatong** [comment]
想请教大家一下使用场景，除了微信小程序和H5、RN之外，大家有需要适配其他小程序吗，如果有的话，一般都是哪些小程序高优呢
字节、支付宝、企业微信、快手

**Reply by ranvk** [reply]
支付宝吧，餐饮类和支付类的会有一定需求

**Comment by cuiyiworld** [comment]
想问一下，一些组件适配taro的源码中，一些标签名称还是div，而不是view，这样能适配多端吗？

**Comment by jay-gaojing** [comment]
Calendar 日历组件在小程序中传递startText和endText无效

**Comment by Hanchengluo** [comment]
组件中已经提供了“Card 商品卡片”，能否再提供“Cate 商品分类”？就是左右结构，左边一列是一级分类，右边是对应的二级分类
